backbone/src/main/res/layout/rsb_step_layout_active_step.xml

```diff
@@ -25,4 +25,22 @@
         android:textSize="@dimen/rsb_step_layout_countdown_text_size"
         android:textColor="@color/rsb_countdown_step_layout_text_color"/>
 
+    <ProgressBar
+        style="@style/Base.Widget.AppCompat.ProgressBar"
+        android:id="@+id/rsb_active_step_layout_progress"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:padding="@dimen/rsb_padding_large"
+        android:indeterminate="true"
+        android:visibility="gone"/>
+
+    <android.support.v7.widget.AppCompatImageView
```

Comment:
Do you need to specify this a AppCompatImageView?  In the docs it says it will automatically be used if you specify ImageView:  https://developer.android.com/reference/android/support/v7/widget/AppCompatImageView.html